src/main/java/lotto/domain/place/Rank.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain.place;
+
+import java.util.Arrays;
+import java.util.function.BiPredicate;
+
+public enum Rank implements BaseRank {
+  FIRST(FIRST_COUNT_OF_MATCH, 2_000_000_000, (countOfMatch, matchBonus) -> countOfMatch == FIRST_COUNT_OF_MATCH),
+  SECOND(SECOND_COUNT_OF_MATCH, 30_000_000, (countOfMatch, matchBonus) -> countOfMatch == SECOND_COUNT_OF_MATCH && matchBonus == true),
+  THIRD(THIRD_COUNT_OF_MATCH, 1_500_000, (countOfMatch, matchBonus) -> countOfMatch == THIRD_COUNT_OF_MATCH && matchBonus == false),
```

Comment:
다음과 같이 해볼 수도 있을 것 같아요! 😄 
```suggestion
  SECOND(SECOND_COUNT_OF_MATCH, 30_000_000, (countOfMatch, matchBonus) -> countOfMatch == SECOND_COUNT_OF_MATCH && matchBonus),
  THIRD(THIRD_COUNT_OF_MATCH, 1_500_000, (countOfMatch, matchBonus) -> countOfMatch == THIRD_COUNT_OF_MATCH && !matchBonus),
```